src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php

```diff
@@ -305,6 +305,30 @@ public function iClickTheToolbarButton($itemTitle)
     }
 
     /**
+     * @When I click the tab item :tabTitle
+     */
+    public function iClickTheTabItem($tabTitle)
+    {
+        $this->clickByTitle('.column-navigation .item-text', $tabTitle);
+
+        $tabItems = $this->getSession()->getPage()->findAll('css', '.tabs-container ul li');
+
+        /* @var NodeElement $tabItem */
+        foreach ($tabItems as $tabItem) {
+            $element = $tabItem->find('named', array('content', $tabTitle));
+            if ($element && $element->isVisible()) {
+                // Click and wait for the ajax request.
+                $element->click();
+                $this->waitForAjax(self::MEDIUM_WAIT_TIME);
+
+                return;
+            }
+        }
+
+        throw new ElementNotFoundException($this->getSession(), null, 'css', '.tabs-container ul li');
```

Comment:
I would make `.tabs-container ul li` a variable, otherwise the selector may change in the future but the error message may not.